components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -1313,7 +1313,7 @@ private static void buildOpenIDConnectAuthenticationConfiguration(
             fedIdp.setDefaultAuthenticatorConfig(oidcAuthnConfig);
         }
 
-        Property[] properties = new Property[10];
+        Property[] properties = new Property[11];
```

Comment:
why it gets the clientId constant from Facebook? Shall we get it from OIDC authenticator similar to client secret?